ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel/Wallets.hs

```diff
@@ -35,22 +37,29 @@ import Ariadne.Wallet.Cardano.Kernel.Internal
   (PassiveWallet, walletKeystore, wallets)
 import qualified Ariadne.Wallet.Cardano.Kernel.Keystore as Keystore
 import Ariadne.Wallet.Cardano.Kernel.PrefilterTx (PrefilteredUtxo)
-import Ariadne.Wallet.Cardano.Kernel.Types (WalletId(..))
+import Ariadne.Wallet.Cardano.Kernel.Types (WalletId(..), AccountId(..))
 import Ariadne.Wallet.Cardano.Kernel.Util (getCurrentTimestamp)
 
 import Test.QuickCheck (Arbitrary(..))
 
 
 data CreateWalletError =
-      CreateWalletFailed HD.CreateHdRootError
+      CreateWalletFailed HD.CreateHdRootError  
```

Comment:
Just one more minor thing. Please configure your editor to strip trailing whitespace.